pages/translations/en/LC_MESSAGES/messages.po

```diff
@@ -1104,7 +1107,7 @@ msgid "…while trying to load the search results."
 msgstr ""
 
 #: frontend/templates/layouts/default.j2:129
-msgid "Sadly this page isn't translated yet, but we could use some help doing so. See the <a href="/documentation/guides-and-tutorials/contribute/translations">translation guide</a> on how you can contribute."
+msgid "Sadly this page isn't translated yet, but we could use some help doing so. See the <a href=\"/documentation/guides-and-tutorials/contribute/translations\">translation guide</a> on how you can contribute."
```

Comment:
are the backslashes a problem here?

Reply:
I dont think so? How do you mean?

Reply:
Was wondering if this will later endup in the html. Not an expert when it comes to PO files.

Reply:
ah, no, it all renders correctly